### PR TITLE
Member toValue as hash

### PR DESCRIPTION
**Pull request by smizell**
This uses what @danielgtaylor has converts it to a hash.

See #75 

cc @pksunkara

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Unreleased
+
+- Add a `.toValue()` method to member elements which returns a tuple of the
```
Comment (pksunkara) [inline CHANGELOG.md]: This description needs to be changed.

**Comment by pksunkara** [comment]
Just a small minor thing.

File: CHANGELOG.md (hunk 1 above)
Comment (smizell) [inline CHANGELOG.md]: Great catch

**Comment by smizell** [comment]
Fixed and squashed the commit

**Pull request merged**